packages/spark-core/components/_promo.scss

```diff
@@ -0,0 +1,36 @@
+// ==================================================================
+// Promo
+// ==================================================================
+.sprk-c-Promo {
+  max-width: $promo-max-width;
+  margin: 0 auto;
+  border: $promo-border;
+}
+
+.sprk-c-Promo__title {
+  font-weight: $font-weight-display-four;
+}
+
+.sprk-c-Promo__subtitle {
+  font-weight: $font-weight-body-one;
+}
+
+.sprk-c-Promo__icon {
+  text-align: center;
+  align-self: center;
+}
+
+.sprk-c-Promo__image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+
+  @media (max-width: $promo-breakpoint) {
+    width: 100%;
```

Comment:
The promo image is being set to 100% and then reset again to 100%. Is this intentional?